locations/spiders/nuvolari_it.py

```
@@ -0,0 +1,38 @@
+            "markers"
+        ]
+
+    def post_process_item(self, item, response, location):
+        item["branch"] = item.pop("name")
+        item["street_address"] = merge_address_lines(item.pop("street"))
+        item["country"] = "IT"
+        if item.get("postcode"):
+            item["postcode"] = item["postcode"].strip()
+
+        item["opening_hours"] = OpeningHours()
+        for day, hours in zip(DAYS_3_LETTERS_FROM_SUNDAY, location["schedule"]["openingHours"]):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)